addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -47,7 +47,7 @@ export class DiscussCoreCommon {
                 author: this.store.odoobot,
                 body: markup(body),
                 id: lastMessageId + 0.01,
-                is_note: true,
+                subtype_id: this.store.mt_comment_id,
```

Comment:
This was renamed to `mt_comment`, but isn't note supposed to be `mt_note` instead?